lib/ui/views/patcher/patcher_viewmodel.dart

```diff
@@ -161,6 +162,40 @@ class PatcherViewModel extends BaseViewModel {
     return text;
   }
 
+  String getCurrentVersionString(BuildContext context) {
+    return '${FlutterI18n.translate(
+      context,
+      'appSelectorCard.currentVersion',
+    )}: v${selectedApp!.version}';
+  }
+
+  Future<void> searchSuggestedVersionOnWeb(BuildContext context) async {
+    final String search = FlutterI18n.translate(
+      context,
+      'appSelectorCard.search',
+    );
+    final String apk = FlutterI18n.translate(
+      context,
+      'appSelectorCard.apk',
+    );
+    final String version = FlutterI18n.translate(
+      context,
+      'appSelectorCard.version',
+    );
+    final String suggestedVersion =
+        _patcherAPI.getSuggestedVersion(selectedApp!.packageName);
```

Comment:
What if the suggested version is Any/All? that doesn't make sense in a web search